.travis.yml

```diff
@@ -2,7 +2,6 @@
 language: ruby
 sudo: false
 rvm:
-  - 1.9.3
```

Comment:
Do we have to drop `1.9.3` with this PR? The issue is that it will have to be a major version update in this case

Reply:
> Do we have to drop 1.9.3 with this PR?

Yes: https://travis-ci.org/RubyMoney/money/jobs/415723861

I'm even can't build `ruby-1.9.3-p551`. It's totally unsupported version since 23 Feb 2015: https://www.ruby-lang.org/en/news/2015/02/23/support-for-ruby-1-9-3-has-ended/

I'm OK with a major version update.